includes/diagnostics.php

```diff
@@ -6,15 +6,24 @@
 $dropin = $this->validate_object_cache_dropin() && ( ! defined('WP_REDIS_DISABLED') || ! WP_REDIS_DISABLED );
 
 $info[ 'Status' ] = $this->get_status();
-$info[ 'Redis Version' ] = $this->get_redis_version() ?: 'Unknown';
+
+if (defined( 'WP_REDIS_CLIENT' ) && WP_REDIS_CLIENT === 'predis' && defined( 'WP_REDIS_CLUSTER' )) {
+    $info[ 'Redis Version' ] = 'Not supported';
+} else {
+    $info[ 'Redis Version' ] = $this->get_redis_version() ?: 'Unknown';
```

Comment:
Wouldn't this just default to `Unknown`? Is this condition really necessary.

Reply:
Original judgement
```
$info[ 'Redis Version' ] = $this->get_redis_version() ?: 'Unknown';
```
It may not work, so I made a change, in fact, it does have problems in my environment! And not in line with the general way of judgment! The format of ternary logic judge is:
```
boolean?true:false
```
So the part of true should not be empty